scripts/variantstore/wdl/extract/Dockerfile

```diff
@@ -21,12 +21,14 @@ RUN apk update && apk upgrade
 RUN python3 -m ensurepip --upgrade
 
 # Add any generally useful Alpine packages here.
-RUN apk add --no-cache jq
+RUN apk add --no-cache perl jq
```

Comment:
what does "perl" here do?

Reply:
Installs perl a once heavily used language (even at the Broad) now largely relegated to the dust heap.